### PR TITLE
Update some links and texts for pelican-elegant theme

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

File: pelicanconf.py [hunk 1]
```diff
@@ -90,58 +90,20 @@
 MAILCHIMP_FORM_ACTION = u'empty'
 
 # SMO
-TWITTER_USERNAME = u'talham_'
+TWITTER_USERNAME = u''
 FEATURED_IMAGE = SITEURL + '/theme/images/apple-touch-icon-152x152.png'
 
 # Legal
-SITE_LICENSE = u'<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"> onCrash="Reboot();"</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="http://oncrashreboot.com" property="cc:attributionName" rel="cc:attributionURL">Talha Mansoor</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.'
+SITE_LICENSE = u'<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"> Pelican-Elegant Documentation"</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="http://oncrashreboot.com" property="cc:attributionName" rel="cc:attributionURL">Talha Mansoor</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/deed.en_US">Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.'
```
Comment (talha131) [inline pelicanconf.py]: href needs an update.

Comment (talha131) [reply pelicanconf.py]: Creative commons license needs to be upgraded from 3.0 to 4. Silverhook has mentioned it in the issues.

File: pelicanconf.py [hunk 2]
```diff
@@ -90,58 +90,20 @@
...
 
 # SEO
-SITE_DESCRIPTION = u'My name is Talha Mansoor \u2015 a software engineer who gets things done. I am talha131 at Github and @talham_ at twitter. I design and build software products for iOS and OSX. I work on Jump Desktop which is a remote desktop application for iOS, OSX and Android. This is my personal blog.'
+SITE_DESCRIPTION = u'This is the documentation website for Pelican-Elegant theme originally created by Talha Mansoor'
```
Comment (talha131) [inline pelicanconf.py]: I think this should be enough.

> Documentation website for Pelican-Elegant theme originally created by Talha Mansoor

File: pelicanconf.py (hunk 1 above)
Comment (talha131) [reply pelicanconf.py]: @iranzo Do you think this issue is relevant here

https://github.com/Pelican-Elegant/documentation/issues/18

Will your PR close the #18?

If so then please change it to

> "Elegant theme documentation is licensed under a CC-BY-SA-4.0 license."

File: pelicanconf.py (hunk 2 above)
Comment (iranzo) [reply pelicanconf.py]: ACK on next push

File: pelicanconf.py (hunk 1 above)
Comment (iranzo) [reply pelicanconf.py]: ACK on next push, and also closes #18

**Comment by talha131** [comment]
Thank you @iranzo. Waiting for you to close, last review left.

File: pelicanconf.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 # -*- coding: utf-8 -*- #
 from __future__ import unicode_literals
 
-AUTHOR = u'Talha Mansoor'
-SITENAME = u"""<span style="color:black;">onCrash</span> <span style="color:darkblue;">=</span> <span style="color:#AA1032;">'reboot();'</span>"""
+AUTHOR = u'Pelican Elegant Team'
+SITENAME = u"Pelican-Elegant Documentation"
```
[the remark this answers is not in the document]
Comment (iranzo) [reply pelicanconf.py]: Submitted in next push

**Comment by iranzo** [comment]
@talha131 submitted, waiting for your review

**Pull request merged**